server/model.js

```diff
@@ -23,16 +26,17 @@ module.exports = {
   },
   writeNewComment: (params, callback) => {
     const {
-      songId, profilePic, username, message, postedAt, songTime, followers,
+      songid, stringid, profilepic, username, message, postedat, songtime, followers
     } = params;
-    console.log('these are the params', params);
-    const createCommentQuery = `INSERT INTO comments (songId, profilePic, username, message, postedAt, songTime, followers)
-                                VALUES  ( "${songId}","${profilePic}","${username}","${message}","${postedAt}","${songTime}","${followers}")`;
-    connection.query(createCommentQuery, (err, newComment) => {
+    const createCommentQuery = `INSERT INTO comments (songid,profilePic,username,message,postedat,songtime,followers)
+                                VALUES  (${songid},'${profilepic}','${username}','${message}','${postedat}','${songtime}',${followers})`;
```

Comment:
Will you be inserting stringid as well?

Reply:
No, stringId as been removed after bench marking whether a string song field was faster that a number song filed. The answer is, it doesn't matter as long as the field is indexed.